src/odemis/driver/semnidaq.py

```diff
@@ -103,11 +103,11 @@
 # by frequent active/inactive status.
 AFTER_SCAN_DELAY = 0.1  # s
 
-# Rough duration of the AI/AO buffer, during which the command loop is not checked.
+# Rough duration of the AI/AO "chunk", during which the command loop is not checked.
```

Comment:
What is the underlying problem and how is chunking going to resolve this (I'm not familiar with the original problem)? Maybe nice to explain in detail in a Software Knowledge transfer at some point.

Reply:
I guess you answered it in the other comment already!